perllib/FixMyStreet/App/Controller/Contact.pm

```diff
@@ -197,6 +197,22 @@ sub prepare_params_for_email : Private {
     my $base_url = $c->cobrand->base_url();
     my $admin_url = $c->cobrand->admin_base_url;
 
+    my $user = $c->cobrand->users->find( { email => $c->stash->{em} } );
+    if ( $user ) {
+        $c->stash->{user_admin_url} = $admin_url . '/users/' . $user->id;
+        $c->stash->{user_reports_admin_url} = $admin_url . '/reports?search=' . $user->email;
+
+        my $user_latest_problem = $c->cobrand->problems->search({
+            user_id => $user->id
```

Comment:
Probably only want to include visible states here? Guess might be of interest if latest report is hidden already, maybe...

Reply:
Feels like we shouldn’t link to anything here that the user themselves can’t access. So that would preclude hidden reports.